### ISSUE TITLE
Specify eslint location

### ISSUE THREAD

**Issue by jglapa** [doits/pronto-eslint_npm#1]
Would it be possible to pass the location of eslint to your plugin?
My problem is that the eslint package in my project is not a global package. 
I'm not a ruby programmer but I was able to test this by changing the `run_eslint` method to 
```ruby
Dir.chdir(@_repo_path) do
   JSON.parse(`./web-app/node_modules/.bin/eslint #{Shellwords.escape(patch.new_file_full_path.to_s)} -f json`)
end
```
and got my analysis finally running successfully.
The question would be: how can I pass such a parameter? 
You change path each time to repo_path but in my case all the FE code is in web-app directory thus I've added that explicitly to the path.

PS.
I've builded and installed the gem locally for testing purposes but I was getting the following error:
`uninitialized constant Shellwords (NameError)` after adding require `'shellwords'` it went away.

**Comment by doits** [comment]
I've added a commit to require `shellwords`.

I'm not sure yet how to configure `eslint` location and whether we really need this. A workaround is to update `PATH` variable before running it, so in your case something like:

```
export PATH = "/full/path/to/web-app/node_modules/.bin/:$PATH"
pronto run ...
```

**Comment by doits** [comment]
Please take a look at latest commits in master. I added the option to specify the location (see README). If this works for you I will push a new version to rubygems.

**Comment by doits** [comment]
Released with v0.7.1.

**Comment by jglapa** [comment]
I just tested it. Works great! Thanks @doits. 

I don't want to commit this file so I'm each time running this:
```
echo "eslint_executable: '$PWD/node_modules/.bin/eslint'" > .pronto_eslint_npm.yaml
```
 when running on CI server. 
Perhaps another improvement would be the ability to pass those setting via env variables.
